EPHHeatRelatedIllness changes as on nov 22

shapateriya opened the pull request.

shamimansari1988 reviewed (commented):
Looks Good.

krishnaswamypradeep reviewed (approved):
Thanks Sudhisha. Looks good.

shapateriya commented:
I've addressed all the comments on https://github.com/datacommonsorg/data/pull/1123 for the import 
EPHHeatRelatedIllness. Since it's outdated, I've closed it and created a new https://github.com/datacommonsorg/data/pull/1162 reflecting the changes. Please review: https://github.com/datacommonsorg/data/pull/1162
